compiler-plugin/src/main/java/io/ballerina/lib/np/compilerplugin/PromptAsCodeCodeModificationTask.java

```diff
@@ -254,6 +259,27 @@ public FunctionCallExpressionNode transform(FunctionCallExpressionNode functionC
             getTypeSchema(typeSymbol.get(), typeMapper, modifierData.typeSchemas);
             return functionCallExpressionNode;
         }
+
+        private boolean isNotNPCallCall(FunctionCallExpressionNode functionCallExpressionNode,
```

Comment:
```suggestion
        private boolean isNotNPCall(FunctionCallExpressionNode functionCallExpressionNode,
```

Reply:
I think the function name should `isNpCall`

Reply:
Then we end up always checking the inverse when calling it.